R/build_lookup.R

```diff
@@ -62,6 +62,41 @@ parse_imgt_fasta <- function(infile) {
   imgt_list
 }
 
+#' Extract gene names from a reference CSV for a given species
```

Comment:
```suggestion
#' Extract gene names from a reference MiXCR CSV
```
I would be a little more specific and concise

Reply:
Made the change in most recent commit

https://github.com/seshadrilab/tcrconvertr/blob/8f63d9743f10aafb4770e70d09c7897f9c47f522/R/build_lookup.R#L64-L66